doc/faq.rst

```diff
@@ -100,12 +100,31 @@ See :ref:`adding_graphical_models`.
 Will you add GPU support?
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-No, or at least not in the near future. The main reason is that GPU support
-will introduce many software dependencies and introduce platform specific
-issues. scikit-learn is designed to be easy to install on a wide variety of
-platforms. Outside of neural networks, GPUs don't play a large role in machine
-learning today, and much larger gains in speed can often be achieved by a
-careful choice of algorithms.
+A limited but growing :ref:`list of scikit-learn estimators
+<array_api_supported>` can already run on GPUs if the input data is provided
+as a PyTorch or CuPy array and if scikit-learn has been configured to accept
+such inputs as explained in :ref:`array_api`.
+
+Most estimators that rely on NumPy for their computationally intensive operations
+can be considered for Array API support and therefore GPU support.
+
+However, not all scikit-learn estimators are amenable to running on GPUs via
+the Array API for fundamental algorithmic reasons. For instance, tree-based
+models currently implemented with Cython in scikit-learn are fundamentally not
+array-based algorithms. Other algorithms such as k means or k nearest neighbors
+rely on array-based algorithms but cannot be efficiently implemented without
+fusing consecutive operations to avoid introducing performance killing memory
+access to unnecessarily large intermediate arrays. Fusing such operations
+cannot be expressed via the Array API for the foreseeable future. Adding
+efficient GPU support to these estimators would require designing and adopting
+a more flexible extension system for scikit-learn. This possibility is being
+considered in the following GitHub issue (under discussion):
+
+- https://github.com/scikit-learn/scikit-learn/issues/22438
+
+In any case, the `scikit-learn` package itself will not add any mandatory
```

Comment:
```suggestion
The `scikit-learn` package itself will not add any mandatory
```

IMHO "In any case" doesn't add anything/just adds more words. We could go even further and turn the sentence around to "No mandatory GPU-specific software dependencies will be added to the `scikit-learn` package to ensure that it remains lightweight and easy to install." - this puts the important message at the start of the sentence.